### DISCUSSION TITLE
<How to Earn Community Badges on GitHub: Tips and Strategies?>

### DISCUSSION THREAD

**Discussion by MohSensei** [community/community#122629] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello GitHub Community!

I’m interested in learning more about earning community badges on GitHub. Could you share insights or strategies on how to engage effectively with the community to achieve these badges? Any specific contributions or activities that are particularly valuable?

**Comment by jrobin11** [comment]
Earning community badges on GitHub involves engaging with the platform in various ways that contribute positively to the community. Here are some general strategies and specific activities that can help you earn badges:

Contributing to Open Source Projects: This is one of the most impactful ways to earn badges. Contributing can involve submitting pull requests to fix bugs, add features, or improve documentation.
Participating in Discussions: Engage in discussions within repositories or join broader conversations in GitHub discussions forums. Providing thoughtful insights and helpful answers can earn you recognition.
Reviewing Code: Offer to review others' pull requests. This not only helps the project but demonstrates your willingness to collaborate and contribute your expertise.
Creating or Contributing to Issues: If you find bugs or have feature requests, opening issues in repositories is another way to get involved. Make sure your reports are clear, detailed, and helpful.
Maintaining a Repository: If you have a project that others find useful, maintaining it actively and responding to community interactions can also lead to badges.
Educational Content: Creating tutorials or educational content related to projects on GitHub can earn badges, especially if they're endorsed or highlighted by the community or GitHub itself.
Participation in Special Events: GitHub often hosts events like Hacktoberfest or CodeSpaces challenges. Participating in these events can also help you earn badges.
Being Active Regularly: Regular activity over time is often recognized, so consistently contributing can lead to earning long-term participation badges.
Remember, the key to earning these badges is genuine, consistent, and valuable engagement with the community. Each badge has specific criteria, so it's also a good idea to read up on what each badge requires and track your progress towards meeting those criteria.

**Reply by ebndev** [reply]
Hi @jrobin11, thanks for wanting to participate in GitHub Discussions! 

Please note: we’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by ebndev** [comment]
Hi @MohSensei,

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
